apps/web/modules/auth/signup/components/signup-form.tsx

```diff
@@ -139,7 +139,11 @@ export const SignupForm = ({
         }
 
         const errorMessage = getFormattedErrorMessage(createUserResponse);
-        toast.error(errorMessage);
+        if (errorMessage === "User with this email already exists") {
```

Comment:
This is a client component, meaning there is the server action (API) `createUserAction` that returns in their payload if a user with the same email address already exists.
So an attacker could just use this server action / API endpoint to enumerate through the email addresses and the fix is just a cosmetic fix in the UI, not solving the problem. Please fix :-)